src/main.cpp

```diff
@@ -369,7 +372,7 @@ int __fastcall myBattleOnProcess(T *This) {
   if (switchByKey && checkKey(switchKey)) {
     selfCifChangedTime = std::chrono::system_clock::now();
     if (!lastPress) {
-      if (!isNetplay) {
+      if (!isNetplay && !checkNoModKeys()) {
```

Comment:
Could you move `checkNoModKeys()` to `if (switchByKey && checkKey(switchKey)) {`, making it `if (switchByKey && checkKey(switchKey) && checkNoModKeys()) {`, so that it is consistent with L401 (L404) and might be cleaner? Or is there a reason not to do this?

Reply:
I put it there in case it would affect stuff outside of toggling cif state, but if you think it's fine then I'll change it.

Reply:
It is fine. It only affects toggling cif state